sdk/communication/azure-communication-callingserver/azure/communication/callingserver/_callingserver_client.py

```diff
@@ -57,9 +65,11 @@ def __init__(
                 "You need to provide account shared key to authenticate.")
 
         self._endpoint = endpoint
+        self._authentication_policy = get_authentication_policy(
+            endpoint, credential)
         self._callingserver_service_client = AzureCommunicationCallingServerService(
             self._endpoint,
-            authentication_policy=get_authentication_policy(endpoint, credential),
+            authentication_policy=self._authentication_policy,
```

Comment:
let's keep it as current.

Reply:
we need to use _authentication_policy in content_download API as well, so assign value in constructor